CHANGELOG.md

```diff
@@ -2,45 +2,69 @@
 
 ## v6.0.0-beta.6 - 2017-08-24
 
+### Breaking change
+
+* Make invalid argument handling to -File and -Command consitent and make exit codes consistent with Unix standards (#4573)
```

Comment:
Only the error message is consistent, the arguments are handled differently by design